pyproject.toml

```diff
@@ -7,7 +7,7 @@ authors = [
     { name = "Mozilla Release Engineering", email = "release+python@mozilla.com" }
 ]
 readme = "README.rst"
-requires-python = ">=3.9"
+requires-python = ">=3.11"
```

Comment:
Shouldn't this be in client/pyproject.toml?

Reply:
(and remove black's target-version there)

Reply:
Oh I didn't think about checking if it was published or not... Will bump